docs/resources/project_policy.md

```diff
@@ -41,8 +41,8 @@ resource "env0_project_policy" "example" {
 - **continuous_deployment_default** (Boolean) Redeploy on every push to the git branch default value
 - **disable_destroy_environments** (Boolean) Disallow destroying environment in the project
 - **include_cost_estimation** (Boolean) Enable cost estimation for the project
-- **number_of_environments** (Number) Max number of environments a single user can have in this project, 0 indicates no limit
-- **number_of_environments_total** (Number) Max number of environments in this project, 0 indicates no limit
+- **number_of_environments** (Number) Max number of environments a single user can have in this project, `null` indicates no limit
```

Comment:
@away168 these files are automatically generated, you need to change the source in `resource_project_policy.go`